packages/protocol/lib/compatibility/verify-bytecode.ts

```diff
@@ -129,7 +128,7 @@ const dfsStep = async (queue: string[], visited: Set<string>, context: Verificat
     implementationAddress = ensureLeading0x(context.libraryAddresses.addresses[contract])
   } else {
     const proxyAddress = await context.registry.getAddressForString(contract)
-    if (proxyAddress === ZERO_ADDRESS) {
+    if (proxyAddress === '0x0000000000000000000000000000000000000000') {
```

Comment:
though the code we use constants for these - for consistency it would make sense to do the same here